Firestore/src/V1beta1/TransactionOptions/PBReadOnly.php

```diff
@@ -0,0 +1,84 @@
+<?php
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
This file is here because I forgot to add it in #5843, and for some reason our tests didn't catch it :-/